application/src/main/java/org/opentripplanner/transit/service/TimetableRepository.java

```diff
@@ -166,10 +166,10 @@ public TimetableRepository() {
    */
   public void index() {
     if (index == null) {
-      LOG.info("Index transit model...");
+      LOG.info("Index timetable repository...");
       // the transit model indexing updates the site repository index (flex stops added to the stop index)
```

Comment:
Seems like this comment could use an update too. It's unclear to me what value it adds.

Reply:
Fixed in f40626194bfc67e2bfdac64f41eb2d69d881d397